CMakeLists.txt

```diff
@@ -104,6 +104,14 @@ endif()
 option(ZSTD_FROM_INTERNET "Download and use libzstd from the Internet" ${ZSTD_FROM_INTERNET_DEFAULT})
 find_package(zstd 1.1.2 REQUIRED)
 
+option(REDIS_STORAGE_BACKEND "Enable redis secondary storage")
+if(REDIS_STORAGE_BACKEND)
+  find_package(PkgConfig)
+  pkg_check_modules(HIREDIS REQUIRED hiredis>=0.13.3)
+  include_directories(${HIREDIS_INCLUDEDIR})
+  add_definitions(${HIREDIS_CFLAGS_OTHER})
+endif()
```

Comment:
If we'd bump-up minimum CMake to 3.6 we could use:
```cmake
pkg_check_modules(HIREDIS REQUIRED IMPORTED_TARGET hiredis>=0.13.3)
...
target_link_libraries(ccache_lib PUBLIC PkgConfig::HIREDIS)
```
and omit everything else.

Reply:
Yes, I reverted that in order to stay compatible.